ztp/siteconfig-generator/siteConfig/siteConfigBuilder_test.go

```diff
@@ -341,6 +341,135 @@ func Test_getExtraManifestTemplatedRoles(t *testing.T) {
 	}
 }
 
+func Test_mergeExtraManifests(t *testing.T) {
```

Comment:
These are great tests, but the ones testing the `MergeManifests` function should be in `siteConfigHelper_test.go` instead.

Reply:
The ones testing the MergeManifests function have been moved to siteConfigHelper_test.go.